docs/application/dotnet/samples/watch/overview.md

```diff
@@ -57,6 +57,13 @@ for TD:
 			</ul></p>
 			</td>
 		</tr>
+		<tr>
+			<td><img alt="" height="180" src="media/w63analogwatch.png" width="180"/></td>
+			<td>
+                        <p><strong>(W) AnalogWatch</strong> [In progress]</p>
+			<p>This sample application demonstrates how to create a watchface for the wearable device using ElmSharp and how to use the Badge APIs.</p>
```

Comment:
> Updated. Badge APIs is correct

Question. Isn't "Badge API" correct?

Reply:
I had a question because of the previous sentence:
Is it Badges API or Badge APIs?
For more information, see https://developer.tizen.org/development/api-references/native-application?redirect=https://developer.tizen.org/dev-guide/3.0.0/org.tizen.native.mobile.apireference/group__BADGE__MODULE.html.

Please verify.
In Native API reference, https://developer.tizen.org/development/api-references/native-application?redirect=https://developer.tizen.org/dev-guide/4.0.0/org.tizen.native.mobile.apireference/group__BADGE__MODULE.html
It is written as "Badge APIs". 
Please correct me if I am wrong. So that we can be consistent throughout. 
Thank you.

Reply:
I've found only 2 APIs within sample page.
For consistency, I've fixed them to "API"

Reply:
OK. Thank you.